Deeploy/Targets/Snitch/TileConstraints/ReshapeTileConstraint.py

```
@@ -0,0 +1,143 @@
+            replacements["size"].append(outSize)
+
+            # For reshape, we need to map output cube to input cube
+            # Calculate flat index range for output cube
+            outOffset = 0
+            outStrides = []
+            stride = 1
+            for dim in reversed(outputShape):
+                outStrides.insert(0, stride)
+                stride *= dim
+
+            for i, (off, dim) in enumerate(zip(cube.offset, cube.dims)):
```

Ruff (0.14.14)

[warning] Loop control variable `dim` not used within loop body

(B007)